lib/builders/bedrock-builder.ts

```diff
@@ -0,0 +1,53 @@
+import { BlueprintBuilder } from '../stacks';
+import * as utils from "../utils";
+import * as addons from '../addons';
+import * as spi from '../spi';
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class BedrockBuilder extends BlueprintBuilder {
+
+    /**
+     * This method helps you prepare a blueprint for setting up EKS cluster with 
+     * usage tracking addon
+     */
+    public static builder(): BedrockBuilder {
```

Comment:
aren't we supposed to add a bedrock team here ootb? e.g. take those props with the function and add the team. Or another alternative is to add a method like addBedrockTeam() to make it explicit. 
The way it stands now, there is no differentiator for this builder for the customers.